### PR TITLE
fix: global/quantity-input.php input closing tag contained within $labelledby IF statement

### LINKED ISSUE #23280: global/quantity-input.php input closing tag contained within $labelledby IF statement
**Describe the bug**
Pull request #22333 adds an IF statement in the _global/quantity-input.php_ template to check if `$labelledby` is not empty before outputting the `aria-labelledby` attribute, however the closing tag for the `<input>` tag is contained within this IF block, therefore if `$labelledby` is empty, the `<input>` tag will be left open.

### PR DESCRIPTION
**Pull request by parmarhardip**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #23280 

### How to test the changes in this Pull Request:

1. Add the PR to a publicly accessible site
2. Run the W3C validator against a single product https://validator.w3.org/

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: includes/class-wc-frontend-scripts.php [hunk 1]
```diff
@@ -355,7 +355,7 @@ public static function load_scripts() {
 		if ( is_cart() ) {
 			self::enqueue_script( 'wc-cart' );
 		}
-		if ( is_cart() || is_checkout() || is_account_page() ) {
+		if ( is_cart() || is_checkout() || is_account_page() || is_product() ) {
```
Comment (mikejolley) [inline includes/class-wc-frontend-scripts.php]: Why was this included?

File: templates/global/quantity-input.php [hunk 2]
```diff
@@ -43,6 +43,8 @@ class="<?php echo esc_attr( join( ' ', (array) $classes ) ); ?>"
 			inputmode="<?php echo esc_attr( $inputmode ); ?>"
 			<?php if ( ! empty( $labelledby ) ) { ?>
 			aria-labelledby="<?php echo esc_attr( $labelledby ); ?>" />
+			<?php } else { ?>
```
Comment (mikejolley) [inline templates/global/quantity-input.php]: Instead of adding an else, you'd just need to move the above `/>` out of the if statement.

**Review by mikejolley — CHANGES_REQUESTED** [review]

File: templates/global/quantity-input.php (hunk 2 above)
Comment (parmarhardip) [reply templates/global/quantity-input.php]: @mikejolley I've updated the condition as per your suggestion :)

File: includes/class-wc-frontend-scripts.php (hunk 1 above)
Comment (parmarhardip) [reply includes/class-wc-frontend-scripts.php]: Removed that. Please [check](https://github.com/woocommerce/woocommerce/pull/23290/commits/826c88a4161548c29d7058fc131c7866801c6a48).

**Comment by mikejolley** [comment]
Thanks, the code changes look good 👍 

I have however just noticed we have a separate PR which removes this check entirely which looks to be a better overall solution to the reported issue (https://github.com/woocommerce/woocommerce/pull/23186)

Thanks for contributing though :)
